src/lib/containers/RssFeedCards.tsx

```diff
@@ -35,13 +40,18 @@ export default class RssFeedCards extends React.Component<RssFeedCardsProps, Rss
   
   componentDidMount() {
     this._isMounted = true
-    const { url } = this.props
-    fetch(url)
+    const { url, lockedFacet } = this.props
+    const lockedFacetValue = lockedFacet?.value
+    const tagPath = lockedFacetValue ? `/tag/${lockedFacetValue.replace(' ', '-')}` : ''
```

Comment:
If a lockedFacet is provided, then the value is used as a tag to filter the items shown.